### ISSUE TITLE
Consolidating Tags

### ISSUE THREAD

**Issue by metastange** [wdetdigital/wdet-conversion#82]
@wdet30 to explore potential plugins to merge duplicative tags.

**Comment by wdet30** [comment]
@metastange, here is the best plugin I can find for this solution. Would you please check this over and let me know if you would like me to deploy.

https://wordpress.org/plugins/term-management-tools/

**Comment by metastange** [comment]
Can you tell me how to access Tags and Categories on the wdetcms site? That doesn't appear to be an option for me. Want to take a look to better acquaint myself there before making this call. Screenshot of menu attached for reference in case I missed it:
![Screen Shot 2021-12-01 at 4 16 34 PM](https://user-images.githubusercontent.com/85897115/144315305-d8c3101a-2331-4329-a085-cd1dac0ab159.png)

**Comment by wdet30** [comment]
@metastange here you go: https://wdetcms.wdet.org/wp-admin/edit-tags.php?taxonomy=post_tag

**Comment by metastange** [comment]
Got it, thanks. Yes, it looks like this will work.

**Comment by artlogic** [comment]
@dorothyhernandez is this something we still need?

**Comment by dorothyhernandez** [comment]
@artlogic Yes. I am not sure what to do on my end so please let me know what I can do to facilitate this.

**Comment by artlogic** [comment]
@dorothyhernandez I think @wdet30 can help you get started. Please reach out to him.

**Comment by dorothyhernandez** [comment]
@wdet30 what was the solution to this? Unless I'm missing something, I'm still seeing deviations of tags on the same topic, e.g. #covid, #Covid-19, COVID, COVID19, Covid 19
